tests/Operation/ChangeStreamFunctionalTest.php

```diff
@@ -0,0 +1,252 @@
+<?php
+
+namespace MongoDB\Tests\Operation;
+
+use MongoDB\ChangeStream;
+use MongoDB\ChangeStreamIterator;
+use MongoDB\Client;
+use MongoDB\Collection;
+use MongoDB\Driver\BulkWrite;
+use MongoDB\Driver\ReadPreference;
+use MongoDB\Exception\ResumeTokenException;
+use MongoDB\Operation\Aggregate;
+use MongoDB\Operation\ChangeStreamCommand;
+use MongoDB\Operation\DatabaseCommand;
+use MongoDB\Operation\InsertOne;
+use MongoDB\Tests\CommandObserver;
```

Comment:
A bunch of these appear to no longer be used in this file:

 * ChangeStreamIterator
 * BulkWrite
 * ReadPreference
 * ResumeTokenException (not required for `@expectedException`)
 * Aggregate
 * InsertOne
 * IteratorIterator
 * stdClass